source/_integrations/pvpc_hourly_pricing.markdown

```diff
@@ -0,0 +1,99 @@
+---
+title: PVPC Hourly Pricing
+description: Instructions on how to set up the ComEd Hourly Pricing sensor in Home Assistant.
+logo: ree_esios.svg
+ha_category:
+  - Energy
+ha_release: '0.107'
+ha_iot_class: Cloud Polling
+ha_quality_scale: platinum
+ha_config_flow: true
+ha_codeowners:
+  - '@azogue'
+---
+
+This sensor uses the official API to get the hourly price of electricity in Spain from https://www.esios.ree.es/en/pvpc.
+
+Specifically, it shows the current __active energy invoicing price (FEU)__ in €/kWh, 
+which is the energy term hourly price applied in the consumers' electrical bill 
+with a contracted power not exceeding 10 kW and which are under the PVPC 
+(Voluntary Price for Small Consumer).
+
+It includes the energy term of the access tolls, the charges and the production cost. It does not include taxes.
+The hourly prices are the same throughout the Spanish territory regardless of the time zone.
+
+It can be set up via the integrations panel in the configuration screen.
+
+<iframe src="https://www.esios.ree.es/en/embed/active-energy-invoicing-price-pvpc" width="100%" height="608"></iframe>
+
+More information available at http://www.cnmc.es/en/ and http://www.omie.es/en/
+
+## Configuration
+
+To configure PVPC Hourly Pricing, you can set it up via the integrations panel in the configuration screen.
+
+Set a name for the price sensor (default is `sensor.pvpc`) and select one of the three tariffs, 
+corresponding to your contracted rate, according to the number of billing periods per day 
+(one / peak + valley / shifted peak + valley).
+
+- 1 period: `normal`, for the "Default PVPC tariff, (2.0 A)".
+- 2 periods: `discriminacion`, for the "Efficiency 2 periods (2.0 DHA)", with 10h peak interval from 11:00 UTC to 21:00 UTC.
```

Comment:
Spelling error `discriminacion` -> `discrimination` ?
```suggestion
- 2 periods: `discrimination`, for the "Efficiency 2 periods (2.0 DHA)", with 10h peak interval from 11:00 UTC to 21:00 UTC.
```

Reply:
Not really :)

I'll give you some context, so you could help me to decide what to do:

- This was a custom sensor for a long time (without config flow), and totally localized in Spanish for Spain, as it pulls the local price for electricity.
- The only decision to make in config is to select one of the three tariffs:
  * _Normal_ := technically called "2.0 A", normal in English, default rate if the consumer does nothing: it's the interesting one for energy suppliers, but not for consumers :)
  * _Discriminación_ := "2.0 DHA", discrimination in English, which is the default one in the integration, as it is usually the best for home consumers.
  * _Coche eléctrico_ := "2.0 DHS", electric car rate.
- When I was doing the PR (which needs to be de-localized, and in English) I doubted using the technical labels for tariffs, but the potential user in Spain wouldn't recognize them easily, probably. So I leave the old ones, just **slugified strings of the original spanish recognizable names**.

As these config labels are not translatable, and the potential user for the integration is _at least located_ in Spain, I thought it was not too bad to leave them in Spanish (as the main PR also includes the Spanish string translations), **but I could change that in both PR's in a bit**, so, what do you think is the best approach?

1. Leave as it is now, with slugified spanish recognizable labels
2. Translate those labels to English, also recognizable ("discrimination" + "electric_car")
3. Use the technical codes (2.0 A/DHA/DHS), the least recognizable IMO, but with the new integration, there is sufficient help in the UI, so it's no big deal.

Reply:
I get that it was Spanish (Español ex novia), nevertheless, IMHO, we should keep Home Assistant English.

Reply:
> Español ex novia

nice :)

> we should keep Home Assistant English.

So 2nd option, right? 

(Using 'normal' / 'discrimination' and 'electric_car' as internal tags, here and inside main PR )

Reply:
I would say that is the right way to go, yep! 👍